daemon/graphdriver/driver.go

```diff
@@ -112,6 +113,13 @@ type Driver interface {
 	DiffDriver
 }
 
+// LayerGetter is an interface to get the path to a layer on the host.
```

Comment:
This could just be defined in `layer/layer_windows.go` if that is the only place it is used